.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 12.13.0
```

Comment:
Do we want to use this version of nodejs or the latest version (13.3.0)?

Reply:
Overall I approve of this though so whatever version gets picked has my approval.

Reply:
Good point, @samcdavid . I'll change this to 13.3.